packages/app/src/components/AppBase/AppBase.tsx

```diff
@@ -31,6 +32,19 @@ import ConfigUpdater from '../Root/ConfigUpdater';
 import { SearchPage } from '../search/SearchPage';
 import { settingsPage } from '../UserSettings/SettingsPages';
 
+export const TitleSetter = () => {
+  const config = useApi(configApiRef);
+  const titleFromConfig = config.getOptionalString('app.title');
+
+  React.useEffect(() => {
+    if (titleFromConfig) {
+      document.title = `Welcome back! | ${titleFromConfig}`;
```

Comment:
Welcome Back is a title just set on the home page, right? I don't think we should hard code that value in the AppBase.